stories/queryWizard/utils.js

```diff
@@ -0,0 +1,26 @@
+import F from 'futil-js'
+import _ from 'lodash/fp'
+
+let differentFirst = (firstCase, normalCase) => (acc, i, list) =>
+  i === 0
+    ? _.iteratee(firstCase)(acc, i, list)
+    : _.iteratee(normalCase)(acc, i, list)
+
+let hasPunctuation = F.testRegex(/.*(?=[.!?]$)/)
+
+let punctuate = (punctuation = '.') =>
+  F.when(F.overNone([_.isEmpty, hasPunctuation]), F.append(punctuation))
```

Comment:
I'm not sure I understand why we need all of this. Why not just have periods in the description strings? 🤔

Reply:
Uh... "just in case" 😅

I wanted to make double dog sure that stitched-together field and type descriptions would look nice and accessible in as many situations as possible -- but, yeah, I guess improperly-formatted description text is trivially preventable as far as edge cases go, so this may have been a little overboard